sympy/core/random.py

```diff
@@ -0,0 +1,190 @@
+"""When you need to use random, import it from here so there is only
+one generator working for SymPy."""
+
+import random as _random
+random = _random.Random()
+randint = random.randint
+randrange = random.randrange
+choice = random.choice
+shuffle = random.shuffle
+uniform = random.uniform
+
+class Random():
```

Comment:
@asmeurer @oscarbenjamin @jksuom , this Random is just faking python's `random.Random`; the user will not be actually creating a new rng thread, they will just be changing the one and only rng seed. Is it a good idea to have only a single SymPy rng thread?

Reply:
I think it should be possible to seed all of sympy with a single seed e.g.:
```
from sympy.core.random import seed
seed(1)
```
Also I remember an issue somewhere about `import sympy` altering the state of the global random. I think this might still do that.

Reply:
If you could test that, please let me know the results.

A `seed` function has been added.

Reply:
> I remember an issue somewhere about `import sympy` altering the state of the global random. I think this might still do that.

```python
>>> from random import random,seed
>>> seed(1)
>>> random()
0.13436424411240122
>>> seed(1);random()
0.13436424411240122
>>> seed(1)
>>> import sympy
>>> random()
0.13436424411240122
```
It appears not to do so, based on the above.